after_commit_everywhere.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "isolator", "~> 0.7"
   spec.add_development_dependency "pry"
+  spec.add_development_dependency "nokogiri", "~> 1.13.3"
```

Comment:
`nokogiri == 1.12.2` (the version in the lock file) failed to build on my M1 Mac.